code/datums/diseases/advance/symptoms/heal.dm

```diff
@@ -513,18 +513,13 @@ im not even gonna bother with these for the following symptoms. typed em out, co
 				playsound(get_turf(M), 'sound/effects/splat.ogg', 50, 1)
 				if(prob(60))
 					if(tetsuo && prob(15))
-						if(A.affected_mob.job == "Clown")
```

Comment:
can you not remove the organ spawns from the symptom? read the code you're removing, please?